documentation/library_guide/dynamic_selection_api/fixed_resource_policy.rst

```diff
@@ -0,0 +1,236 @@
+fixed_resource_policy
+#####################
+
+Always returns the same resource selection. ``fixed_resource_policy`` 
+is designed for two primary scenarios: (1) debugging the use of dynamic selection 
+and (2) special casing a dynamic selection capable application for a specific 
+resource is always best on a specific platform.
+
+.. code:: cpp
+
+  namespace oneapi::dpl::experimental {
+  
+    template<typename Backend=sycl_backend> 
+    class fixed_resource_policy {
+    public:
+      // useful types
+      using resource_type = typename Backend::resource_type;
+      using wait_type = typename Backend::wait_type;
+      
+      class selection_type {
+      public:
+        fixed_resource_policy<Backend> get_policy() const;
+        resource_type unwrap() const;
+      };
+      
+      // constructors
+      fixed_resource_policy(deferred_initialization_t);
+      fixed_resource_policy(::std::size_t offset=0);
+      fixed_resource_policy(const std::vector<resource_type>& u,  
+                            ::std::size_t offset=0);
+  
+      // deferred initializers
+      void initialize(::std::size_t offset=0);
+      void initialize(const std::vector<resource_type>& u, 
+                      ::std::size_t offset=0);

```

Comment:
do we want to use `offset` as the public terminology? How do you feel about `index`?

Reply:
This is what is currently in the code. I think the documentation should match. But we might change the code, then the documentation.